go/ngql/README.md

```diff
@@ -0,0 +1,39 @@
+# Noms GraphQL
+
+An experimental bridge between noms and [GraphQL](http://graphql.org/)
+
+# Status
+
+ * All Noms types are supported except
```

Comment:
Cool. Can you say a little about whether any of these are fundamental issues, vs just not-implemented-yet?